tests/cupyx_tests/scipy_tests/sparse_tests/test_index.py

```diff
@@ -0,0 +1,234 @@
+import unittest
+
+import cupy
+from cupy import testing
+
+import pytest
+
+
+@testing.parameterize(*testing.product({
+    'format': ['csr', 'csc'],
+    'density': [0.1, 0.4, 0.9],
+    'dtype': ['float32', 'float64', 'complex64', 'complex128'],
+    'n_rows': [25, 150],
+    'n_cols': [25, 150]
+}))
+@testing.with_requires('scipy')
+@testing.gpu
+class TestIndexing(unittest.TestCase):
```

Comment:
Since this `_run()` function spreads multiple lines, I'd just comment instead of writing suggestions 😅 Generally, `cupy.testing` will do the device-to-host transfer for you, so you don't have to call `get()` or do sync yourselves.